README.md

```diff
@@ -17,6 +17,9 @@ If you would like to contribute with code, please follow the instructions below:
 *   [Guidelines for IVIM code contribution](doc/guidelines_for_contributions.md)
 *   [Guidelines to creating a test file](doc/creating_test.md) 
 
```

Comment:
I think here we could start a header with 
## How to use
and give some explanation on how to look at the Git and how to use it. Also we could refer to the wrapper explanation that Ivan wrote too.